packages/sdk/src/messageExtension/teamsMsgExtTokenResponse.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { TokenResponse } from "botframework-schema";
+
+/**
+ * Token response provided by Teams Bot SSO prompt
+ */
+export interface TeamsMsgExtTokenResponse extends TokenResponse {
```

Comment:
Do we want to use abbreviation in the name? Not sure whether `MessagingExtensionTokenResponse` can be better.
cc @MuyangAmigo

Reply:
Ideally use full name creates more clarity. `MessageExtensionTokenResponse` would be better. (MessageExtension instead of MessagingExtension)